ng-jcrop.js

```diff
@@ -166,6 +167,10 @@
         $scope.getShrinkRatio = function(){
             var img = $('<img>').attr('src', $scope.mainImg[0].src)[0];
 
+            if(ngJcropConfig.jcrop.maxWidth > img.width || ngJcropConfig.jcrop.maxHeight > img.height){
```

Comment:
What happens if `imgWidth < maxWidth` but `imgHeight > maxHeight`. Won't the image still get shrunken?